### DISCUSSION TITLE
Request: Downloadable app for mobile devices

### DISCUSSION THREAD

**Discussion by ghost** [linuxserver/Heimdall#702] [Ideas]
Request for a downloadable app from iOS and Android similar to how Home Assistant perform it, this would then also allow for devices to be automatically signed into the desired account.

I understand this does take quite a bit of work.

**Comment by pybe** [comment]
With android at least you can visit the site and then just choose add to home screen from the menu in chrome.

**Comment by ghost** [comment]
This is something I currently do but the layout isn't as friendly as it could be.

I was thinking more of a future feature possibility.

**Comment by dontobi** [comment]
I also currently use Heimdall via shortcut on the homescreen. However, an app would make sense for me, because I can automatically set up my VPN through it (via Tasker).
